climpred/classes.py

```diff
@@ -908,16 +908,16 @@ def remove_seasonality(
             >>> HindcastEnsemble.remove_seasonality(seasonality="month")
             <climpred.HindcastEnsemble>
             Initialized:
-                SST      (init, lead, member) float64 -0.2349 -0.216 ... 0.6476 0.6433
+                SST      (init, lead, member) float64 -0.2392 -0.2203 ... 0.618 0.6136
```

Comment:
Unsure why this changed

Reply:
Is this function dependent on any algorithms from scikit-learn? We've needed to adjust some outputs from time to time in `xclim` due to changes there.